src/main/resources/assets/admin/common/js/schema/content/ContentTypeSummary.ts

```diff
@@ -118,6 +122,10 @@ export class ContentTypeSummary
         return this.metadata;
     }
 
+    getAllowChildContentType(): string[] {
+        return this.allowChildContentType;
+    }
```

Comment:
Instead of this we need `isMediaChildContentAllowed`:

```Typescript
isMediaChildContentAllowed(): boolean {
  return this.allowChildContent &&
      this.allowedChildContentTypes?.some((name: string) => new ContentTypeName(name).isDescendantOfMedia());
}
```

Reply:
Disagree with this part: let's keep ContentTypeSummary.ts as a data class without non-related logic. Child check is moved to ContentTypesHelper.ts in studio